app.js

```diff
@@ -1,18 +1,34 @@
 const express = require("express");
-const { getTopics, getArticleById } = require("./controllers/get.controllers");
+const { getTopics } = require("./controllers/topic.controllers");
+const {
+  getArticleById,
+  patchArticleById,
+} = require("./controllers/article.controllers");
 
 const app = express();
 
+app.use(express.json());
+
 app.get("/api/topics", getTopics);
 
 app.get("/api/articles/:article_id", getArticleById);
 
+app.patch("/api/articles/:article_id", patchArticleById);
+
 app.all("/*", (req, res) => {
   res.status(404).send({ msg: "No such endpoint!" });
 });
 
 app.use((err, req, res, next) => {
-  res.status(404).send({ msg: err.msg });
+  if (err.status === 404) {
```

Comment:
You can make this and the error handler below more dynamic and DRY-er but having res.status(err.status).send({msg: err.msg}) - then you can use the same function for both of these, AND you don't have to keep writing middleware for each custom error.